helios_auth/auth_systems/password.py

```diff
@@ -104,9 +104,10 @@ def get_auth_url(request, redirect_url = None):
   return reverse(password_login_view)
     
 def get_user_info_after_auth(request):
-  user = request.session['password_user']
-  del request.session['password_user']
-  user_info = user.info
+  from helios_auth.models import User
+  user_id = request.session['password_user_id']
+  user = User.objects.get(id=request.session['password_user_id'])
```

Comment:
Better to use `User.get_by_type_and_id` and specify type to be `password`, so that this doesn't open up some possible cross-user-type security risk.